trin-types/src/node_id.rs

```diff
@@ -0,0 +1,24 @@
+use serde::{Deserialize, Serialize};
+use serde_hex::{SerHex, StrictPfx};
+use std::ops::Deref;
+
+type RawNodeId = [u8; 32];
+
+/// Discv5 NodeId
+// TODO: Wrap this type over discv5::NodeId type
```

Comment:
We're more likely to make sure this gets followed up on, if we move it out of a TODO and into a Ready issue in the tracker.